### DISCUSSION TITLE
docs/getting-started/installing/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#275] [Page comments]
# docs/getting-started/installing/

https://scratchaddons.com/docs/getting-started/installing/

**Comment by Pokebrouserkat** [comment]
how to install on safari? (scratch is "blocked by CORS policy" in chrome D:)

**Reply by CST1229** [reply]
you can't

**Reply by Pokebrouserkat** [reply]
D: safari supports extensions from the app store, is there even a chance of putting it there?

**Comment by Eranchick** [comment]
Как скачать на yandex?

**Reply by Eranchick** [reply]
А, понял, как в гугл хром

**Comment by Baster013** [comment]
Can you make the Opera one?

**Reply by Baster013** [reply]
nwm

**Comment by ghost** [comment]
This is not possible on Safari, Correct?

**Reply by CST1229** [reply]
Yes, SA is not available for Safari. (And it probably won't be anytime soon, because Safari doesn't support some of the things SA uses, and there's a $100/_year_ fee for putting things on the App Store.)

**Comment by BenjiSkye15** [comment]
how to install on opera GX?

**Reply by Samq64** [reply]
Try the Chrome version.

**Comment by jqualt3r** [comment]
is it possible to install it onto the scratch app?

**Reply by CST1229** [reply]
No, but most of the addons are available in https://desktop.turbowarp.org

**Reply by BroJac5246** [reply]
> No, but most of the addons are available in https://desktop.turbowarp.org

Note that this doesn't apply if you're on Android or ChromeOS. In that case, use the TurboWarp website, unless you're on ChromeOS and feel comfortable installing the Linux desktop version in the Debian VM, which is relatively advanced.

**Comment by Bruh-King** [comment]
Please release for opera gx

**Reply by BroJac5246** [reply]
Hi! Opera GX is based on Chromium, the same open-source framework that Chrome and Edge run on. As a result, it should have proper support for Chrome extensions from the Chrome Web Store. Have you tried installing that way?

**Comment by Bruh-King** [comment]
yes it does not work

**Reply by WorldLanguages** [reply]
What doesn't work?

**Reply by BroJac5246** [reply]
> What doesn't work?

https://github.com/ScratchAddons/website-v2/discussions/275#discussioncomment-9757446

**Comment by hoangkongu** [comment]
But where should I write the npm install command?

**Reply by Samq64** [reply]
In the terminal but you shouldn't install it that way unless you have to.

**Comment by LDRA772** [comment]
Is there a safari Version?

I just wanna use it on my MacBook using the safarii version :(

**Reply by CST1229** [reply]
no, you'll have to use another browser

**Comment by scratcher-guinea-pig-snuggles** [comment]
Hi

**Reply by scratcher-guinea-pig-snuggles** [reply]
How do I make a moving thumbnail?

**Reply by scratcher-guinea-pig-snuggles** [reply]
Meow

**Comment by scratcher-guinea-pig-snuggles** [comment]
Hi!

**Comment by makbpobre-spec** [comment]
gg
